TicTacToe2/GameManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Script;
+using TicTacToe2.Model;
+
+namespace TicTacToe2
+{
+    public class GameManager
+    {
+        // Start is called before the first frame update
+  
+        private bool _waitingForPlayerAction = true;
+
+        private TicTacToeMap _map;
+    
+        void Start()
+        {
+            _map = new TicTacToeMap();
+            EventController.CallEvent("mapChange");
+        }
+
+        // Update is called once per frame
+        void Update()
```

Comment:
Custom game object with awake, start, update, destroy,  
maybe children object for a tree